package.json

```diff
@@ -82,13 +82,15 @@
 		"jsdom": "^3.1.2",
 		"mocha": "^2.2.4",
 		"jscs": "~1.6.2",
-		"jshint": "^2.6.0"
+		"jshint": "^2.6.0",
+		"nan": "^1.8.4"
 	},
 	"engines": {
-		"node": ">=0.10"
+		"node": ">=0.12"
```

Comment:
Actually, you're right -- i just view the code again and seems like i've deleted all the >0.12-dependent code, thanks ```nan```